acceptance-amazon-rds-sync/backup-table-field.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Local directory
+DIR=$(dirname "$0")
+TABLE_NAME="$1"
+PRIMARY_KEY="$2"
+FIELD_NAME="$3"
+WHERE="$4"
+
+# Include common RDS tasks
+source $DIR/config
+
+if [ "$TABLE_NAME" == '' ]; then
+  echo
+  echo "ERROR - table name is missing" >&2
+  exit 1
+fi
+
+if [ "$PRIMARY_KEY" == '' ]; then
+  echo
+  echo "ERROR - primary key is missing" >&2
+  exit 1
+fi
+
+if [ "$FIELD_NAME" == '' ]; then
+  echo
+  echo "ERROR - field name is missing" >&2
+  exit 1
+fi
+
+backup_filename=$TABLE_NAME"_"$PRIMARY_KEY"_"$FIELD_NAME".field_backup.sql"
+query="SELECT CONCAT('UPDATE $TABLE_NAME SET $FIELD_NAME = ', QUOTE($FIELD_NAME), ' WHERE $PRIMARY_KEY = ', $PRIMARY_KEY, ';') FROM $TABLE_NAME $WHERE"
```

Comment:
Yes. `QUOTE` function is handling the escape:
https://dev.mysql.com/doc/refman/5.0/en/string-functions.html#function_quote